src/features/flows/nRF9151/SimCard.tsx

```diff
@@ -13,40 +13,51 @@ import { Next } from '../../../common/Next';
 
 const SIMCard = () => (
     <Main>
-        <Main.Content heading="Plug in SIM card">
-            <p>There are two pre-activated SIM cards in the kit:</p>
-            <br />
-            <ol className="tw-list-outside tw-list-disc tw-pl-4">
-                <li>
-                    <Link
-                        label="SIM card from Onomondo"
-                        href="https://onomondo.com/go/nordic-dev-kit/#form"
-                        color="tw-text-primary"
-                    />{' '}
-                    with 10 MB free data.
-                    <br />
-                    If you register the SIM card, you will get additional 40 MB
-                    to use within the SIMs lifetime. In addition, you will get a
-                    two-month trial access to Onomondos real-time network
-                    insight tool.
-                </li>
-                <br />
-                <li>
-                    <Link
-                        label="SIM card from Wireless Logic"
-                        href="https://www.wirelesslogic.com/simclaim/nsctrial/"
-                        color="tw-text-primary"
-                    />{' '}
-                    with 5 MB free data for 6 months.
-                    <br />
-                    If you register the SIM card within the trial period, you
-                    will get additional 45 MB to use within 5 years.
-                </li>
-            </ol>
-            <br />
-            <p>
-                Select the SIM card that is supported in your area (check the
-                links), plug it in, and it will work out of the box.
+        <Main.Content
+            heading="Plug in SIM card"
+            subHeading="There are two pre-activated SIM cards included that work out-of-the-box*"
```

Comment:
This needs to be here, because you've added "Check coverage", so this foreshadows the purpose of those buttons:
```suggestion
            subHeading="There are two pre-activated SIM cards included that work out of the box (as long as the SIM card has coverage in your area):"
```